extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -29,9 +30,46 @@ public function getContentRelatedTo($title, $limit=5, $ignore=null) {
 		}
 
 		$api = $this->relatedContentApi();
-		
+
 		try {
 			$filteredRelatedContent = $api->getRelatedContentFromEntityName($title, $limit + 1, "true");
+            if (is_null($filteredRelatedContent)) {
+                WikiaLogger::instance()->warning("getRelatedContentFromEntityName failed to retrieve recommendations", [
```

Comment:
the `Loggable` trait, which this class uses, adds `$this->warning`, so importing the logger and using `WikiaLogger::instance` is unnecessary (though it all ends up going to the same place)